tests/integration_tests/security/test_seccomp.py

```diff
@@ -66,14 +66,15 @@ def _get_basic_syscall_list():
     return json
 
 
-def _run_seccompiler(json_data):
+def _run_seccompiler(json_data, basic=False):
```

Comment:
nit: I d rename the title of the commit to
```
tests: add test for seccompiler's "--basic" arg
```
so that all your commit titles have the same format.